doc/ref_arch/openstack/chapters/appendix-a.md

```diff
@@ -0,0 +1,25 @@
+[<< Back](../../openstack)
+# Appendix A - Guidelines
+<p align="right"><img src="../figures/bogo_ifo.png" alt="scope" title="Scope" width="35%"/></p>
+
+## Table of Contents
+* [A.1 Guidance for Vendors](#A.1)
+* [A.2 RA Transitional Generations](#A.2)
+* [A.3 TBC](#A.3)
+* [A.4 TBC](#A.4)
+
+<a name="A.1"></a>
+## A.1 Guidance Vendors
+
+<a name="A.2"></a>
+## A.2 RA Transitional Generations
+
+CNF Generation definitions are defined in [RM Appendix](../../../ref_modle/chapters/appendix-a#A.3)
```

Comment:
"VNF Generation ..." not "CNF Generation ..."

Reply:
should be fixed now